doc-Monitoring_Alerts_and_Reporting/topics/Integrating_Prometheus_Alerts.adoc

```diff
@@ -0,0 +1,73 @@
+
+It is possible to view ongoing alerts in {product-title_short} and manage their life cycle, including the ability to:
+
+* View active alerts per provider in the alert dashboard.
+* View alert data and related objects.
+* Assign and acknowledge alerts in the alerts list screen.
+
+=== Integrating Prometheus Alerts in {product-title_short}
+
+Complete the following prerequisite steps on the OpenShift Container Platform:
+
+. Deploy and Configure Prometheus on OpenShift; see https://docs.openshift.com/container-platform/3.7/install_config/cluster_metrics.html#openshift-prometheus[Prometheus on OpenShift Container Platform].
+. Add Prometheus to an OpenShift cluster and configure alert definitions. 
+.. See intructions below to define alerts (currently has to be done on the Prometheus side):
++
+----
+$ oc edit configmap -n openshift-metrics prometheus
+# Supported annotations:
+# severity: ERROR|WARNING|INFO. defaults to ERROR.
+# miqTarget: ContainerNode|ExtManagementSystem, defaults to ContainerNode.
+# miqIgnore: "true|false", should ManageIQ pick up this alert, defaults to true.
+  prometheus.rules: |
+    groups:
+    - name: example-rules
+      interval: 30s # defaults to global interval
+      rules:
+      - alert: "Node Down"
+        expr: up{job="kubernetes-nodes"} == 0
+        annotations:
+          miqTarget: "ContainerNode"
+          severity: "ERROR"
+          url: "https://www.example.com/fixing_instructions"
+          message: "Node {{$labels.instance}} is down"
+
+----
++
+. Reload the Prometheus configuration. You can do this by deleting the Prometheus pod, then redeploying with the new configuration.
+
+
+Complete the following procedure to enable Prometheus alerts in {product-title_short}:
+
+. In the {product-title_short} user interface, navigate to menu:Control[Explorer], then click *Alert Profiles* in the accordion menu.
++
+[NOTE]
+====
+Creating a profile is not required since both profiles, Node and Provider, are created automatically during the installation.
+====
++
+. Click to expand *Node Alert Profiles*, then click *Prometheus Node Profile*. 
+. Click *Configuration*, then click *Edit assignments for this Alert Profile*. Assign the profile to *The Enterprise* from the *Assign To* list.
+. Click *Save*.
+. Click to expand *Provider Alert Profiles*, then click *Prometheus Provider Profile*. 
+. Click *Configuration*, then click *Edit assignments for this Alert Profile*. Assign the profile to *The Enterprise* from the *Assign To* list.
+. Click *Save*.
++
+[NOTE]
+====
+Alerts that were triggered before assigning the profiles to the enterprise will not appear in {product-title_short} at all.
+====
++
+. Add a Prometheus alerts endpoint.
```

Comment:
This is referring to this:
![ssl_error](https://user-images.githubusercontent.com/3010449/34217078-0c5079fc-e5b3-11e7-879f-0b4d2f144621.png)


(there's a bug there that no longer exists, but that's why I tool the screen shot...)

Reply:
Thanks - I believe this includes the procedure to add OpenShift Containers Provider and the Alerts endpoints? I have updated accordingly.